js/src/chrome/settings/chrome.settings.tooltip.js

```diff
@@ -143,14 +143,10 @@ Wu.Chrome.SettingsContent.Tooltip = Wu.Chrome.SettingsContent.extend({
 		Wu.DomEvent.on(titleInput, 'blur', this.saveTitle, this);
 	},
 
-	// Description
-	initDescription : function () {
```

Comment:
Why deleted? This could perhaps be overwriting the parent function, so it might have a purpose even if empty.  Prefer not to delete functions unless we're SURE they're not needed.  (Also, we like to comment out functions for a while before deleting them. :)